Rearranged Boston Python Workshop and OSCTC nodes in the events page

sunilkumarc opened the pull request:
Hi @shaunagm @ehashman 

I have rearranged the 'Boston Python Workshop' and 'OSCTC' nodes in the events page.
This fixes https://github.com/openhatch/oh-mainline/issues/1602. 

Please review the changes and let me know if any more changes are required.

Thanks,
Sunil

phildini commented on mysite/base/templates/base/events.html:
```diff
@@ -53,36 +53,36 @@
-	  how to participate in open source communities. The workshop is a collaboration with existing
-	  on-campus computing groups so that open source becomes a lasting part of campus culture.
-	</div>
+  <div class="name-of-event">
```
For this block, is github freaking out or did the indentation suddenly change? Indentation should probably match the rest of the block.

sunilkumarc replied:
I just rearranged the two blocks in my sublime text editor. I don't know what happened to the indentation.

Deborah-Digges replied:
Try choosing the "indent using spaces" option at the bottom right of your sublime text editor and set the tab width to 4 spaces. I suspect your tab width is currently set to two spaces.

sunilkumarc replied:
Thanks @Deborah-Digges . It worked! :smile:

sunilkumarc commented:
Hi @phildini,
1. I have fixed the indentation issues.
2. I have squashed the commits into a single commit.

Please check if this fix is eligible for the merge now.

Thanks,
Sunil

Deborah-Digges commented:
This looks good, so I went ahead and merged it. Verified the changes locally:
![screenshot from 2015-04-19 13 09 11](https://cloud.githubusercontent.com/assets/3866405/7218876/7001f7d6-e6a2-11e4-94fb-fdf2d57c6fe5.png)
